docs/common/craft-parts/reference/plugins/rust_plugin.rst

```diff
@@ -174,8 +174,22 @@ To get even better performance, you might want to follow the tips below.
 
 * (Advanced) Perform cross-language LTO. This requires installing the correct version of LLVM/Clang and setting the right environment variables.
   You must know which LLVM version of your selected Rust toolchain is using.
-  For example, Rust 1.71 uses LLVM 16 because you can see it bundles a ``libLLVM-16-rust-1.71.1-stable.so`` file in the ``lib`` directory.
-  In this case, you want to install ``clang-16`` and ``lld-16`` from the Ubuntu archive.
+  You can use ``rustc -vV`` to print out the LLVM version used by the compiler. For example, you can see Rust 1.81 uses LLVM 18.1 because
+  it prints out an output like this:
```

Comment:
```suggestion
  You can use ``rustc -vV`` to check the LLVM version used by the compiler. For example, you can see Rust 1.81 uses LLVM 18.1 because
  it prints an output like this:
```